buildenv/jenkins/variables/defaults.yml

```diff
@@ -52,21 +58,29 @@ linux_ppc-64_cmprssptrs_le:
     8: '/usr/lib/jvm/java-7-openjdk-ppc64el'
     9: '/usr/lib/jvm/adoptojdk-java-ppc64le-80'
     10: '/usr/lib/jvm/adoptojdk-java-ppc64le-90'
```

Comment:
is it a good time to remove the platform shorts from the 8,9,10 while we're here? Same for ppcle

Reply:
It would have been when I was adding jdk10 to the machines.  I'll likely be updating jdk10 with something more current in the next month.  I'll try remember to do that then.